src/components/OptionButtons/OptionButtons.tsx

```diff
@@ -0,0 +1,12 @@
+import SortByButton from '../SortByButton';
+import styles from './OptionButtons.module.css';
+
+function OptionButtons() {
+   return (
+      <div className={styles.optionButtons}>
+         <SortByButton />
+      </div>
+   );
+}
+
+export default OptionButtons;
```

Comment:
In general, this approach is ok.
But what will happen if in the future you will add more option buttons (that you currently don't think about them)?
Will need to go over each one of the buttons and change their style just so you could add the extra button well?
What about if this component was mainly just a wrapper which defines the spacing of this area in relation to other areas, and also the spacing to have between the buttons (like with a gap)?